bin/webpack-helpers.js

```diff
@@ -150,16 +149,17 @@ const stableMainEntry = {
 	'active-filters': './assets/js/blocks/active-filters/index.js',
 	'block-error-boundary':
 		'./assets/js/base/components/block-error-boundary/style.scss',
-	'single-product': './assets/js/blocks/single-product/index.js',
+	cart: './assets/js/blocks/cart-checkout/cart/index.js',
+	checkout: './assets/js/blocks/cart-checkout/checkout/index.js',
 };
 
 const experimentalMainEntry = {
-	cart: './assets/js/blocks/cart-checkout/cart/index.js',
-	checkout: './assets/js/blocks/cart-checkout/checkout/index.js',
+	'single-product': './assets/js/blocks/single-product/index.js',
 };
 
 const mainEntry =
-	process.env.WOOCOMMERCE_BLOCKS_PHASE === 'stable'
+	// env variables are strings, so we compare against a string, so we need to parse it.
+	parseInt( process.env.WOOCOMMERCE_BLOCKS_PHASE, 10 ) < 3
 		? stableMainEntry
```

Comment:
So this means that we still have dead code removal on _releases_ for anything marked experimental right? Might be a good idea to also check that `process.env.WOOCOMMERCE_BLOCKS_PHASE` is defined before using here? If it's undefined won't this cause a ReferenceError (especially since it looks like we no longer define a default for `process.env.WOOCOMMERCE_BLOCKS_PHASE`)?

Reply:
It shouldn’t cause a Reference Error because `process.env` is always defined, `WOOCOMMERCE_BLOCKS_PHASE` not being defined will result in experimental files being generated which is safer than them not being generated right?

Reply:
I am okay with experimental files being built if `WOOCOMMERCE_BLOCKS_PHASE` is not defined, that's not what I'm calling out here. I thought we might have an issue if `parseInt` returned `NaN` but it looks like we won't:

<img width="358" alt="Image 2020-06-02 at 3 32 39 PM" src="https://user-images.githubusercontent.com/1429108/83561766-63156000-a4e6-11ea-85f6-dce73fbcd91e.png">

So all is okay here.